app/javascript/mastodon/features/status/index.jsx

```diff
@@ -173,7 +173,7 @@ const titleFromStatus = status => {
   const prefix = displayName.trim().length === 0 ? username : displayName;
   const text = status.get('search_index');
 
-  return `${prefix}: "${truncate(text, 30)}"`;
+  return `${prefix}: ${text ? `"${ truncate(text, 30)}"` : '(media attached)'}`;
```

Comment:
Hi! All UI text should be translatable (see `intl.formatMessage` calls). In addition, you can use `status.get('media_attachments').size` to know how many attachments the post has.

Reply:
@ClearlyClaire, I added the required `en` and `es-MX` translations, and I translated the status title.